catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/LatLonCoordinateProcessor.java

```diff
@@ -38,12 +37,11 @@
    * @param query the query string with potential coordinate literals present.
    */
   public List<Suggestion> enhanceResults(List<Suggestion> results, String query) {
-    LinkedList<Suggestion> enhanced = new LinkedList<>(results);
     LiteralSuggestion literal = getLatLonSuggestions(query);
     if (literal != null && literal.hasGeo()) {
-      enhanced.addFirst(literal);
+      results.add(0, literal);
     }
-    return enhanced;
+    return results;
```

Comment:
❓ Why have this method return the List when it's not being assigned to anything?